### ISSUE TITLE
Unhandled rejection for local file:dependency

### ISSUE THREAD

**Issue by nwoltman** [tjunnone/npm-check-updates#500]
- [x] I have read the list of [known issues](https://github.com/tjunnone/npm-check-updates#known-issues) before filing this issue.
- [x] I have searched for [similiar issues](https://github.com/tjunnone/npm-check-updates/issues) before filing this issue.

* node version: 10.15.1
* npm version: 6.4.1
* npm-check-updates version: 3.0.5

---------------------------

I have a local dependency in my package.json that looks like this:

```js
{
  "devDependencies": {
    ...
    "eslint-plugin-internal": "file:devtools/eslint-rules",
    ...
  }
}
```

Since upgrading to v3.0.5, I get this error when running `ncu`:

```
Unhandled rejection PackageNotFoundError: Package `eslint-plugin-internal` could not be found
    at packageJson (/usr/lib/node_modules/npm-check-updates/node_modules/package-json/index.js:70:10)
    at processTicksAndRejections (internal/process/next_tick.js:81:5)
```

**Comment by nwoltman** [comment]
Looks like this was caused by #497 since `ncu@3.0.4` seems to work fine.

**Comment by petercunha** [comment]
I don't think it was caused by 529beba. There are several commits that went into `3.0.5`, so my guess is that it was caused by 49e258a ("Replace npm cli calls with package-json").

@stoically could you please look into this issue?

**Comment by 6apxat** [comment]
I confirm this error also occurs with github hosted packages. It started with 3.0.5 update.

**Comment by raineorshine** [comment]
Should be fixed in v3.0.7.
